src/utils/ee-utils.js

```diff
@@ -33,136 +33,192 @@ export const cleanData = (data) =>
         value: f.properties.value,
     }))
 
-export const getEarthEngineValues = async ({
+const getReducedCollection = ({
+    ee,
+    collection,
+    startDate,
+    endDate,
+    reducer,
+}) =>
+    collection
+        .filter(ee.Filter.date(startDate, endDate))
+        [reducer]()
+        .set('system:index', startDate.format('YYYYMMdd'))
+        .set('system:time_start', startDate.millis())
+        .set('system:time_end', endDate.millis())
+
+export const getEarthEngineValues = ({
```

Comment:
Is it on purpose that `getEarthEngineValues = async (...` was switched to `getEarthEngineValues = (...`, async was removed, and will it impact anything?

Reply:
As the function now returns a promise the effect is the same as using async.